thehive/info.json

```diff
@@ -0,0 +1,2241 @@
+        }
+      ],
+      "enabled": ""
+    },
+    {
+      "title": "Create Observable in Alert",
+      "operation": "create_observable_in_alert",
+      "description": "Add an observable to an existing alert.",
+      "category": "investigation",
+      "annotation": "create_observable_in_alert",
+      "output_schema": [
+        {
+          "_id": "",
+          "_type": "",
+          "_createdBy": "",
+          "_updatedBy": "",
+          "_createdAt": "",
+          "_updatedAt": "",
+          "dataType": "",
+          "data": "",
+          "startDate": "",
+          "attachment": {
+            "_id": "",
+            "_type": "",
+            "_createdBy": "",
+            "_updatedBy": "",
+            "_createdAt": "",
+            "_updatedAt": "",
+            "name": "",
+            "hashes": [
+            ],
+            "size": "",
+            "contentType": "",
+            "id": ""
+          },
+          "tlp": "",
+          "tlpLabel": "",
+          "pap": "",
+          "papLabel": "",
+          "tags": [
+          ],
+          "ioc": "",
+          "sighted": "",
+          "sightedAt": "",
+          "reports": {},
+          "message": "",
+          "extraData": {},
+          "ignoreSimilarity": ""
+        }
+      ],
+      "parameters": [
+        {
+          "title": "Alert ID",
+          "name": "alertID",
+          "type": "text",
+          "description": "Specify the ID of the alert for which you want to create an observable in The Hive Platform",
+          "tooltip": "Specify the ID of the alert for which you want to create an observable in The Hive Platform",
+          "visible": true,
+          "editable": true,
+          "required": true
+        },
+        {
+          "title": "Data Type",
+          "name": "dataType",
+          "type": "select",
+          "description": "Select the input type for creating an observable. You can choose either 'Create Observable using Attachment' or 'Create Observable using Plain Text.'",
+          "tooltip": "Select the input type for creating an observable. You can choose either 'Create Observable using Attachment' or 'Create Observable using Plain Text.'",
+          "visible": true,
+          "editable": true,
+          "required": true,
+          "options": [
+            "Autonomous-System",
+            "Domain",
+            "File",
+            "FQDN",
+            "Hash",
+            "IP",
+            "Mail",
+            "Mail_Subject",
+            "Other",
+            "Regexp",
+            "Registry",
+            "Uri_Path",
+            "URL",
+            "User-Agent"
+          ],
+          "onchange": {
+            "Autonomous-System": [
+              {
+                "title": "Data",
+                "name": "data",
+                "type": "text",
+                "description": "Provide the data for the observable that you wish to create and associate with a case in The Hive Platform",
```

Comment:
Please correct description and tooltip for all the parameters "to create and associate with a case" this action is related to alert not for case.